src/main/resources/com/uipath/uipathpackage/UiPathRunJob/help-folderName.html

```diff
@@ -0,0 +1,3 @@
+<div>
+    Specify the folder to run to. If the folder is a modern folder, you will also can use specific user and/or specific machine.
```

Comment:
Super minor clarification:
```suggestion
    Specify the folder to run to. If the folder is a modern folder, you can also target a specific user and/or a specific machine.
```